e-signet/1.8.0/eSignet-signup-mock-UI-Automation/EsignetUi-esqa2-2026-05-07-23-38-T-17-P-9-F-0-S-0-KI-8.html

```
@@ -0,0 +1,1915 @@
+      <tr class="event-row">
+        <td><span class="badge log info-bg">Info</span></td>
+        <td>11:26:13 PM</td>
+        <td>
+          Entered Identifier Value - Verified visibility<details><summary>Locator Details</summary><pre>[[RemoteWebDriver: chrome on windows (a3e3787febcbd62137db5983ee29ce8ee61f6ab1)] -> id: phone]</pre></details>
+        </td>
+      </tr>
+      <tr class="event-row">
+        <td><span class="badge log info-bg">Info</span></td>
+        <td>11:26:13 PM</td>
+        <td>
+          Entered Identifier Value - Entered text: '6793321278'<details><summary>Locator Details</summary><pre>[[RemoteWebDriver: chrome on windows (a3e3787febcbd62137db5983ee29ce8ee61f6ab1)] -> id: phone]</pre></details>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)